packages/paste-core/components/sidebar/src/navigation/types.ts

```diff
@@ -0,0 +1,7 @@
+import type {DisclosurePrimitiveInitialState, DisclosurePrimitveStateReturn} from '@twilio-paste/disclosure-primitive';
+
+export interface SidebarNavigationDisclosureStateReturn extends DisclosurePrimitveStateReturn {
+  [key: string]: any;
```

Comment:
question: does this just mean that any prop can be added to the state return? why does it need to be extended this way

Reply:
Copied from the main Disclosure. I don't recall the prior art reasoning, but just kept it the same.

Reply:
I don't think we need it, Nora and I discussed this for another component, seems redundant in disclosure too